### PR TITLE
Es/delete map modal

### PR DESCRIPTION
**Pull request by ErikSin**
Deletes map.

The delete map button does not belong where it is. This is a temporary stop gap until edit modal is complete.

### CODE REVIEW

File: src/components/DeleteMapModal/index.tsx [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+          {t(msgs.cannot_undo)}
+        </Typography>
+
+        <Typography variant="h3" fontSize={24} align="center">
+          {t(msgs.will_be_deleted)}
+        </Typography>
+
+        <Box display="flex" alignItems="space-between" justifyContent="space-between">
+          <Button
+            color="error"
```
Comment (lightlii) [inline src/components/DeleteMapModal/index.tsx:48]: add `disableElevation` prop

File: src/components/DeleteMapModal/index.tsx [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+    await deleteDoc(doc(db, `groups/${user.uid}/maps`, `${id}`))
+  }
+
+  return (
+    <Dialog open={open}>
+      <Stack display="flex" alignItems="center" justifyContent="center" padding={5} spacing={3}>
+        <DeleteForeverIcon sx={{ fontSize: 150 }} />
+
+        <DialogTitle>
+          <Typography variant="h1" fontSize={32}>
```
Comment (lightlii) [inline src/components/DeleteMapModal/index.tsx]: align center

File: src/components/DeleteMapModal/index.tsx [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+            sx={{
+              borderRadius: 5,
+              display: 'flex',
+              justifyContent: 'space-between',
+              textTransform: 'none',
+              fontWeight: 600,
+              marginRight: 5,
+            }}
+          >
+            <Typography>{t(msgs.delete_button)}</Typography>
```
Comment (lightlii) [inline src/components/DeleteMapModal/index.tsx]: Remove `Typography` component inside buttons or set their component to be a `span` then apply the style props (`fontWeight` etc) to the interior element.

**Review by lightlii — COMMENTED** [review]
looks good, just added a couple of nitpick comments about styling

File: src/components/DeleteMapModal/index.tsx (hunk 1 above)
Comment (ErikSin) [reply src/components/DeleteMapModal/index.tsx:48]: https://github.com/digidem/mapeo-webmaps/pull/83/commits/c25d81fa5131a14f582b6a873081e895cb6e2faf

File: src/components/DeleteMapModal/index.tsx (hunk 2 above)
Comment (ErikSin) [reply src/components/DeleteMapModal/index.tsx]: https://github.com/digidem/mapeo-webmaps/pull/83/commits/c25d81fa5131a14f582b6a873081e895cb6e2faf

File: src/components/DeleteMapModal/index.tsx (hunk 3 above)
Comment (ErikSin) [reply src/components/DeleteMapModal/index.tsx]: https://github.com/digidem/mapeo-webmaps/pull/83/commits/c25d81fa5131a14f582b6a873081e895cb6e2faf

**Comment by ErikSin** [comment]
Lets wait to merge this since the delete button is in a temp spot

**Pull request merged**